GOVERNANCE.md

```diff
@@ -95,7 +95,7 @@ Maintainers are the technical authority for a subproject and are considered lead
 - Regularly attends Kyverno [Maintainers and Community Meetings](https://kyverno.io/community/#community-meetings)
 - Registered for the Kyverno [mailing list](https://groups.google.com/g/kyverno)
 - Create a pull request to add self to `CODEOWNERS` file in at least one (1) repository
-- Attained a minimum of two (3) positive votes from maintainers
```

Comment:
Don't we want all maintainers to agree when voting for a new maintainer ?

Reply:
A majority vote makes sense to me.

Reply:
My feeling is that if one of the maintainers is not ok to onboard a contributor as a new maintainer it should be considered a red flag.
What other projects are doing in this area ?